<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_ui</class>
 <widget class="QMainWindow" name="admin_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>请选择您要进行的操作：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inRenut">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>录入餐厅信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>录入景点信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inRest">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>录入休息区信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chRenut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>160</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>修改餐厅信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chView">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>修改景点信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chRest">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>修改休息区信息</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
